pkg/crdschema/yaml/convert.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yaml
+
+import (
+	"fmt"
+	"reflect"
+
+	yaml2 "gopkg.in/yaml.v2"
```

Comment:
Should we use `sigs.k8s.io/yaml`?
And then we have YamlToJson and JsonToYaml.
https://github.com/kubernetes-sigs/yaml/blob/4e761d0940e599a9569b441f02025fc5981ec2a5/yaml.go#L81-L113

Reply:
we don't need json `[]byte` to yaml `[]byte` or conversely. 

Also note that `sigs.k8s.io/yaml`'s `Marshal/Unmarshal` are not order preserving as they go through JSON which uses maps. yaml.v2 uses `MapSlice` intentionally.

Reply:
Ack